18 ReactHotLoader/webpack.config.js

```diff
@@ -64,6 +62,8 @@ module.exports = {
 	},
 
   plugins:[
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoErrorsPlugin(),
```

Comment:
About noErrorsPlugin, we have to be careful:

https://github.com/gaearon/react-hot-boilerplate/issues/1

https://github.com/gpbl/isomorphic500/issues/81

https://github.com/gaearon/react-transform-boilerplate/issues/122